### ISSUE TITLE
Schedule:list throws error

### ISSUE THREAD

**Issue by StephenB4431** [laravel/framework#40284]
- Laravel Version: 8.78.1
- PHP Version: 8.0.10

### Description

Whilst creating my first scheduled command, trying to list my schedules with 
`php artisan schedule:list`
throws the error:
` DateTime::setTimezone(): Argument #1 ($timezone) must be of type DateTimeZone, null given
at vendor/laravel/framework/src/Illuminate/Console/Scheduling/ScheduleListCommand.php:43`

changing my command to 
`php artisan schedule:list --timezone=Europe/London`
gives:
`DateTime::setTimezone(): Argument #1 ($timezone) must be of type DateTimeZone, string given
at vendor/laravel/framework/src/Illuminate/Console/Scheduling/ScheduleListCommand.php:43`

### Steps To Reproduce:

My schedule:
`$schedule->command('email:expired-licences')->weekdays()->at('08:00')`
My custom command doesn't actually do anything yet.

Eventually I found a solution if I edited ScheduleListCommand.php
changing:
`->setTimezone($this->option('timezone', config('app.timezone')))`
to
`->setTimezone(new DateTimeZone($this->option('timezone', config('app.timezone'))))`
but only if I use the --timezone=Europe/London switch (not sure if this is required), if not then the $ths->option doesn't seem to be accepting the config('app.timezone') as a default.

**Comment by driesvints** [comment]
I can run the above just fine:

```php
    /**
     * Define the application's command schedule.
     *
     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
     * @return void
     */
    protected function schedule(Schedule $schedule)
    {
        $schedule->command('route:list')->weekdays()->at('08:00');
    }
```

<img width="1268" alt="Screenshot 2022-01-07 at 09 18 39" src="https://user-images.githubusercontent.com/594614/148513636-c8ffde59-8510-4f26-a58f-f9ddb1eee209.png">

Please try a support channel:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://web.libera.chat/?nick=laravelnewbie&channels=#laravel)

**Comment by StephenB4431** [comment]
An actual bug - fixed by @royvanv in [commit #926cde9](https://github.com/laravel/framework/commit/926cde9c06ffddec6c3374199ef8f2acbcca7446), hopefully released in next version.
